llvm/lib/Transforms/Instrumentation/NumericalStabilitySanitizer.cpp

```diff
@@ -598,7 +654,14 @@ static GlobalValue *createThreadLocalGV(const char *Name, Module &M, Type *Ty) {
 }
 
 NumericalStabilitySanitizer::NumericalStabilitySanitizer(Module &M)
-    : DL(M.getDataLayout()), Context(M.getContext()), Config(Context) {
+    : DL(M.getDataLayout()), Context(M.getContext()), Config(Context),
+      NsanCopyFns(M, {"__nsan_copy_4", "__nsan_copy_8", "__nsan_copy_16"},
+                  "__nsan_copy_values", 3),
+      NsanSetUnknownFns(M,
+                        {"__nsan_set_value_unknown_4",
```

Comment:
nit: `git clang-format --style=llvm`

Reply:
this command tells me that there are no modified files and the style is correct

Reply:
I've corrected codestyle of previous commits, but these lines stay unchanged